x/axelarnet/keeper/msg_server.go

```diff
@@ -373,6 +373,40 @@ func (s msgServer) RegisterFeeCollector(c context.Context, req *types.RegisterFe
 	return &types.RegisterFeeCollectorResponse{}, nil
 }
 
+// RetryIBCTransfer handles retry a failed IBC transfer
+func (s msgServer) RetryIBCTransfer(c context.Context, req *types.RetryIBCTransferRequest) (*types.RetryIBCTransferResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+	chain, ok := s.nexus.GetChain(ctx, req.Chain)
+	if !ok {
+		return nil, fmt.Errorf("invalid chain %s", req.Chain)
+	}
+
+	if !s.nexus.IsChainActivated(ctx, chain) {
```

Comment:
the id is not linked to a chain atm, someone can just provide any active chain?

Reply:
There's a check below to verify that the chain's IBC path matches the IBC transfer path